main.py

```diff
@@ -33,6 +33,9 @@ def main(args):
         
         # Get a reddit post
         post = ri.get_post()
```

Comment:
Implement logic that throws an exception if there are no reddit posts inside get_post()
Add try/catch here to handle exception and log error